python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_sobol_confidence.py

```diff
@@ -0,0 +1,295 @@
+"""
+Compute Sobol' indices confidence intervals
+-------------------------------------------
+"""
+
+# %%
+# In this example we will estimate confidence intervals for chaos Sobol' indices by bootstrap.
+# Unlike estimation by sampling where the confidence intervals are obtained from the asymptotic distributions,
+# there are no analytical ways to compute Sobol' confidence intervals, so we can fallback to bootstrap.
```

Comment:
```suggestion
# there are no analytical ways to compute Sobol' confidence intervals, so we can fallback to bootstrap.
# To do this, we must simultaneously bootstrap in the input and output samples, so that the input/output mapping is 
# preserved.
# In the example below, we show how to use the :class:`~openturns.BootstrapExperiment` class for this purpose. 
```